src/torchcodec/decoders/_core/video_decoder_ops.py

```diff
@@ -255,6 +256,13 @@ def get_frames_by_pts_in_range_abstract(
     )
 
 
+@register_fake("torchcodec_ns::get_key_frame_indices")
+def get_key_frame_indices_abstract(
+    decoder: torch.Tensor, *, stream_index: int
+) -> List[int]:
```

Comment:
I suspect this will be more useful if we returned a tensor instead of a list. Because tensor are a lot easier to  sub-sample with fancy indexing than lists are, and this function will typically be used in the context of a sampler. E.g.

```py
all_key_frame_indices = decoder._get_key_frame_indices()
NUM_KEY_FRAMES_TO_GET = 10
subset = torch.randint(0, len(all_key_frame_indices), size=(NUM_KEY_FRAMES_TO_GET,))
key_frames_indices = all_key_frame_indices[subset]

```



(Also, I'm assuming that the `vec<int> -> tensor` binding is as cheap or cheaper than the `vec<int> -> list[int]` conversion)